save_load.py

```diff
@@ -20,7 +20,7 @@
                              l.startswith("#")]
 except Exception as e:
     print("couldnt find extra models paths")
-    models_paths = "../saved-models"
+    models_paths = ["../saved-models"]
```

Comment:
good catch. locally though, you should just add the file `models-paths.txt` with the following contents:
```
# first path is the one you always want it to save to, other paths are 
# other options to search in
../saved-models
```
as this list will grow a little when you start working through runai


also, if you could add a comment to the code just above this exception, explaining the importance of the first line in this file (that it is the save location, and the others are just extra search locations), that would be great

Reply:
The comment I should add is something in this lines or am I getting it wrong? I am not sure of what the extra paths are for.
`# the first directory is the save location, and the rest are just extra
# search locations in case there is no model in the first one`

Should I keep the brackets un line 23 anyway right?

Reply:
`# models will be saved specifically in the first path in this file. however, model_explorer.py will use *all* of the paths listed in this file to find models.`